napari/_qt/_tests/test_get_reader_choice.py

```diff
@@ -0,0 +1,91 @@
+from typing import TYPE_CHECKING
+
+from napari._qt.qt_viewer import _get_reader_choice_for_file
+
+if TYPE_CHECKING:
+    from typing import Dict, Optional, Tuple
```

Comment:
I believe you can do these imports on e.g. line 1 and then you don't need to put the types in quotes below (line 18)

Reply:
@nclack I've updated that - hopefully I understood you correctly lol